src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -53,7 +53,7 @@ namespace
             m_tile_size = 64;
             m_sampler_type = 0;
             m_noise_seed = 0;
-            m_enable_noise_seed = false;
+            m_enable_noise_seed = true;
```

Comment:
That variable is actually misnamed: it should be `m_enable_noise_seed_variation`, or even better `m_vary_noise_seed`.